nix/envs/default.nix

```
@@ -0,0 +1,75 @@
+{ src, lib, cachix, fetchFromGitHub }:
+{
```

This may fix the nixpkgs issue:

```suggestion
lib.mapAttrs (_: v: builtins.toString v) {
```

---

works in nixpkgs